apps/nav-veileders-flate/src/components/tiltak/forslag/ForlengDeltakelseForslagDetaljer.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  AktivtForslag,
+  ForlengDeltakelseForslag
+} from '../../../api/data/forslag.ts'
+import { BodyLong, Detail, Heading, HStack, Tag } from '@navikt/ds-react'
+import { EndreDeltakelseType } from '../../../api/data/endre-deltakelse-request.ts'
+import { EndringTypeIkon } from '../EndringTypeIkon.tsx'
+import { formatDateFromString } from 'deltaker-flate-common'
+import { getForslagStatusTypeText } from '../../../utils/displayText.ts'
+
+interface Props {
+  forslag: AktivtForslag
+  forlengDeltakelseForslag: ForlengDeltakelseForslag
+}
+
+export const ForlengDeltakelseForslagDetaljer = ({
+  forslag,
+  forlengDeltakelseForslag
+}: Props) => {
+  return (
+    <div>
+      <HStack gap="2" className="mt-2">
+        <EndringTypeIkon type={EndreDeltakelseType.FORLENG_DELTAKELSE} />
+        <Heading level="6" size="small">
+          Forleng deltakelse
+        </Heading>
```

Comment:
Her er det store muligheter for å trekke ut en komponent for `ForslagDetaljer` men det er sikkert lettere å gjøre når man har flere forslag å jobbe med.

Reply:
ja, absolutt!